pkgs/development/compilers/dotnet/versions/5.0.nix

```diff
@@ -0,0 +1,64 @@
+{ buildAspNetCore, buildNetRuntime, buildNetSdk, icu }:
+
+# v5.0 (current)
```

Comment:
I know this was an issue prior to this PR as well, but it'd be nice if this comment were to be updated to communicate this version is EOL.